back/node_watcher/src/tasks/index.ts

```diff
@@ -26,14 +26,14 @@ import { NomidotTask } from './types';
 export const nomidotTasks: NomidotTask[] = [
   createBlockNumber,
   createCouncil,
-  createSession,
-  createStake,
-  createOfflineValidator,
-  createReward,
-  createEra,
-  createSlashing,
-  createTotalIssuance,
-  createNominationAndValidators,
+  // createSession,
```

Comment:
Those still crash (#233)

Reply:
all of these tasks need to be commented out?

Reply:
That's what the running script currently has, bumping api doesn't help. So unless someone takes a look, things will crash if you uncomment them.. https://github.com/paritytech/Nomidot/pull/236/files#diff-ff11cc905e8168185224695c0855c8c4